sudoku/validate_sudoku.py

```diff
@@ -0,0 +1,119 @@
+# validate sudoku
+# input is a board
+# Return True if board is correct
+# Return False if board is incorrect
+
+
+def validate(board):
+
+    board_len = len(board)
+
+    # # check row numbers for duplicates
+    seen = {}
+    dups = []
+    for row in range(board_len):
+        #print( bo[i])
+        for col in board[row]:
+            
+            if col not in seen:
+                seen[col] = 1
+            else:
+                dups.append(col)
+            
+        seen={}
```

Comment:
Don't need to reset if you just declare at the top of the loop instead of outside the loop.

Reply:
fixed the code